lib/customer.rb

```diff
@@ -20,6 +20,18 @@ def invoices
   end
 
   def transactions
-    repository.find_transactions(id)
+    customer_transactions = invoices.map { |invoice| invoice.transactions }.flatten
```

Comment:
Useless assignment to variable - `customer_transactions`.<br>Line is too long. [83/80]<br>Pass `&:transactions` as an argument to `map` instead of a block.